checker/tests/resourceleak/MultipleMethodParamsMustCallAliasTest.java

```diff
@@ -0,0 +1,89 @@
+import java.io.*;
+import java.net.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.common.returnsreceiver.qual.*;
+
+class MultipleMethodParamsMustCallAliasTest {
+
+  void testMultiMethodParamsCorrect1(@Owning InputStream in1, @Owning InputStream in2)
+      throws IOException {
+
+    ReplicaInputStreams r = new ReplicaInputStreams(in1, in2);
+
+    r.close();
+  }
+
+  void testMultiMethodParamsCorrect2(@Owning InputStream in1, @Owning InputStream in2)
+      throws IOException {
+
+    ReplicaInputStreams r = new ReplicaInputStreams(in1, in2);
+
+    try {
+      in1.close();
+    } catch (IOException e) {
+    } finally {
+      in2.close();
+    }
+  }
+
+  // Is this a bug in checker framework?
```

Comment:
I think this has to be an imprecision somewhere.  The code clearly closes `in1` and `in2` on all paths.  We should open an issue to investigate as a follow up

Reply:
The only difference between this and the previous passing test case is the `catch` block. That definitely shouldn't make a difference, since it's empty! So I suspect there's some imprecision in CFG construction when there is no catch block.

Reply:
I agree there's some imprecision in CFG construction. I remember another [example](https://github.com/Nargeshdb/checker-framework/blob/multipleParamsWithMCA/checker/tests/resourceleak/HBaseReport1.java) that we randomly reported a warning.

Reply:
I opened an issue for this bug https://github.com/typetools/checker-framework/issues/4843